module/VuFind/src/VuFind/Db/Entity/User.php

```diff
@@ -739,7 +740,7 @@ public function getEmailVerified(): ?DateTime
      *
      * @return string[]|\Rbac\Role\RoleInterface[]
      */
-    public function getRoles()
+    public function getRoles(): Traversable|array
```

Comment:
In the IdentityInterface the return type is `iterable`. Is there a reason why you chose `Traversable|array` here?

If not here would be the change:
```suggestion
    public function getRoles(): iterable
```

Reply:
Not sure where that came from, but it wasn't an intentional choice.